tests/js/spec/views/projectEnvironments.spec.jsx

```diff
@@ -6,44 +6,113 @@ import ProjectEnvironments from 'app/views/projectEnvironments';
 
 import EnvironmentStore from 'app/stores/environmentStore';
 
+function mountComponent(isHidden) {
+  const org = TestStubs.Organization();
+  const project = TestStubs.Project();
+  let path = isHidden ? 'environments/hidden/' : 'environments/';
+  return mount(
+    <ProjectEnvironments
+      params={{
+        orgId: org.slug,
+        projectId: project.slug,
+      }}
+      route={{path}}
+    />,
```

Comment:
https://github.com/getsentry/sentry/blob/getsentry-show-hide-environments/tests/js/setup.js#L44-L55

routerContext might be what you want here instead, it's less verbose for this common argument

Reply:
Thanks! Forgot that existed